src/components/accordion/CdrAccordionGroup.vue

```diff
@@ -2,7 +2,7 @@
 import {
   defineComponent, useCssModule, computed, ref, reactive, onMounted, provide,
 } from 'vue';
-import debounce from 'lodash-es/debounce';
+import { debounce } from 'lodash-es';
```

Comment:
We need to destructure the named exports from the `lodash-es` module, which is the module entry point defined in `lodash-es/package.json`. Otherwise, in the context of testing, Vitest whines about not being able to import `lodash-es/debounce` without a `.js` extension.